pegasus/sites.v3/hourofcode.com/views/about_headshots.haml

```diff
@@ -0,0 +1,24 @@
+#headshots{:style=>'width: 900px; padding: 0; margin: 0;'}
+  - people.each_slice(columns) do |group|
+    .headshot_row{:style=>'width: 900px; padding: 0; margin: 0;'}
+      - group.each do |person|
+        - column_width = columns==3 ? '300' : '160'
+        .headshot_column{:style=>'width: ' + column_width + 'px; padding: 15px; margin: 0; float: left;'}
```

Comment:
Obvious improvements could be made here, like extracting CSS, but understood if you just want to make this a duplicate.